tmva/tmva/test/DNN/TestRegressionMethodDL.h

```diff
@@ -0,0 +1,148 @@
+// @(#)root/tmva/tmva/cnn:$Id$
+// Author: Siddhartha Rao Kamalakara, Vladimir Ilievski
+
+/**********************************************************************************
+ * Project: TMVA - a Root-integrated toolkit for multivariate data analysis       *
+ * Package: TMVA                                                                  *
+ * Class  :                                                                       *
+ * Web    : http://tmva.sourceforge.net                                           *
+ *                                                                                *
+ * Description:                                                                   *
+ *      Testing Method DL for Regression                                          *
+ *                                                                                *
+ * Authors (alphabetical):                                                        *
+ *      Siddhartha Rao Kamalakara       <srk97c@gmail.com>   - CERN, Switzerland  *
+ *      Vladimir Ilievski      <ilievski.vladimir@live.com>  - CERN, Switzerland  *
+ *                                                                                *
+ * Copyright (c) 2005-2015:                                                       *
+ *      CERN, Switzerland                                                         *
+ *      U. of Victoria, Canada                                                    *
+ *      MPI-K Heidelberg, Germany                                                 *
+ *      U. of Bonn, Germany                                                       *
+ *                                                                                *
+ * Redistribution and use in source and binary forms, with or without             *
+ * modification, are permitted according to the terms listed in LICENSE           *
+ * (http://tmva.sourceforge.net/LICENSE)                                          *
+ **********************************************************************************/
+
+#ifndef TMVA_TEST_DNN_TEST_REGRESSION_TEST_METHOD_DL_H
+#define TMVA_TEST_DNN_TEST_REGRESSION_TEST_METHOD_DL_H
+
+#include "TFile.h"
+#include "TTree.h"
+#include "TString.h"
+#include "TROOT.h"
+
+#include "TMVA/MethodDL.h"
+#include "TMVA/DataLoader.h"
+#include "TMVA/Factory.h"
+#include "TMVA/Config.h"
+
+
+#include <iostream>
+
+
+/** Testing the entire pipeline of the Method DL, when only a Multilayer Percepton
+ *  is constructed. */
+//______________________________________________________________________________
+void testMethodDL_DNN(TString architectureStr)
+{
+
+   ROOT::EnableImplicitMT(1);
+   TMVA::Config::Instance();
+   
+   TFile *input(0);
+
+   input = TFile::Open("~/Documents/gsoc/root/tree.root", "CACHEREAD");
```

Comment:
Please update this to generate the data when the test is run instead of referencing the (now removed) file.